test/integration-test/src/tests/relocations.rs

```diff
@@ -55,8 +55,28 @@ fn relocate_enum() {
     }
     .build()
     .unwrap();
-    assert_eq!(test.run().unwrap(), 4);
-    assert_eq!(test.run_no_btf().unwrap(), 1);
+    assert_eq!(test.run().unwrap(), 0xBBBBBBBB);
+    assert_eq!(test.run_no_btf().unwrap(), 0xAAAAAAAA);
+}
+
+#[integration_test]
+fn relocate_enum_signed() {
+    let test = RelocationTest {
+        local_definition: r#"
+            enum foo { D = -0x7AAAAAAA };
+        "#,
+        target_btf: r#"
+            enum foo { D = -0x7BBBBBBB } e1;
+        "#,
+        relocation_code: r#"
+            #define BPF_ENUMVAL_VALUE 1
+            value = __builtin_preserve_enum_value(*(typeof(enum foo) *)D, BPF_ENUMVAL_VALUE);
+        "#,
+    }
+    .build()
+    .unwrap();
+    assert_eq!(dbg!(test.run().unwrap()) as i64, -0x7BBBBBBBi64);
+    assert_eq!(dbg!(test.run_no_btf().unwrap()) as i64, -0x7AAAAAAAi64);
```

Comment:
Do you think it would be helpful to put these hex values in constants and document them with comments so that future code readers who find their way in here would have some breadcrumbs to help them understand their origin and significance?

Reply:
Added the following comment:
```
// In the tests below we often use values like 0xAAAAAAAA or -0x7AAAAAAA. Those values have no
// special meaning, they just have "nice" bit patterns that can be helpful while debugging.
```